package.json

```diff
@@ -85,6 +85,7 @@
     "jsdom": "^16.4.0",
     "nyc": "^15.1.0",
     "react-dom": "^16.13.1",
-    "react-test-renderer": "^16.13.1"
+    "react-test-renderer": "^16.13.1",
+    "semantic-release": "^19.0.2"
```

Comment:
we also need all of the semantic-release plugins we're using

Reply:
sorry, this is needed too: https://github.com/liatrio/sparky/blob/main/package.json#L25

Reply:
added!